driver-core/src/main/java/com/datastax/driver/core/querybuilder/Utils.java

```diff
@@ -17,252 +17,107 @@
 
 import java.math.BigDecimal;
 import java.math.BigInteger;
-import java.net.InetAddress;
-import java.nio.ByteBuffer;
-import java.util.*;
+import java.util.List;
 import java.util.regex.Pattern;
 
-import com.datastax.driver.core.DataType;
-import com.datastax.driver.core.utils.Bytes;
+import com.datastax.driver.core.CodecRegistry;
+import com.datastax.driver.core.TypeCodec;
 
 // Static utilities private to the query builder
 abstract class Utils {
```

Comment:
This class is already "aware" of CodecRegistry - but its methods are currently being called with the default instance exclusively.